README.md

```diff
@@ -40,6 +40,8 @@ Below table shows the list of samples available in this repository.
 
 | Folder                           | Description                                |
 |----------------------------------|--------------------------------------------|
+| [`managed-identity-keyvault`](./managed-identity-keyvault) | Sample project using system assigned managed identity to access Key Vault |

```

Comment:
`system assigned` should be `system-assigned`. Check [here](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview#how-does-the-managed-identities-for-azure-resources-work).